dbt_semantic_interfaces/validations/semantic_models.py

```diff
@@ -178,3 +179,43 @@ def _validate_semantic_model(semantic_model: SemanticModel) -> List[ValidationIs
             issues.append(error)
 
         return issues
+
+
+class SemanticModelDefaultsRule(SemanticManifestValidationRule[SemanticManifestT], Generic[SemanticManifestT]):
+    """Checks defaults in semantic models."""
+
+    @staticmethod
+    @validate_safely(whats_being_done="running model validation ensuring the defaults are valid")
+    def validate_manifest(semantic_manifest: SemanticManifestT) -> Sequence[ValidationIssue]:  # noqa: D
+        issues: List[ValidationIssue] = []
+
+        for semantic_model in semantic_manifest.semantic_models:
+            issues.extend(SemanticModelDefaultsRule._validate_default_agg_time_dimension(semantic_model=semantic_model))
+        return issues
+
+    @staticmethod
+    @validate_safely(whats_being_done="checking validity of the semantic model's default agg_time_dimension")
+    def _validate_default_agg_time_dimension(semantic_model: SemanticModel) -> List[ValidationIssue]:
+        issues: List[ValidationIssue] = []
+
+        if semantic_model.defaults is None:
+            return []
+
+        default_agg_time_dimension = semantic_model.defaults.agg_time_dimension
+        assert default_agg_time_dimension, "should not be None"
```

Comment:
Nit: Because of python's wonkiness this assert could fail for other reasons than it being `None`. I think it's safer to specifically check `None`, i.e. `assert default_agg_time_dimension is not None`, simply because if we do run into the insanely weird scenario where `assert default_agg_time_dimension` fails for a reason other than `None`, we don't go the wrong direction when debugging.

Reply:
Sorry, that was a mistake. It should also be optional in the Pydantic implementation.

Reply:
Hmmm. That's a great point. I personally think it should be optional (that then just means the user is required to specify an agg dimension for every measure on a given semantic model)

Reply:
cool, made it Optional!